bigchaindb/crypto/fulfillments/ed25519_sha256.py

```diff
@@ -0,0 +1,266 @@
+import base58
+
+from bigchaindb.crypto.ed25519 import Ed25519VerifyingKey
+from bigchaindb.crypto.fulfillments.base_sha256 import BaseSha256Fulfillment
+from bigchaindb.crypto.buffer import Predictor
+
+
+class Ed25519Sha256Fulfillment(BaseSha256Fulfillment):
```

Comment:
I noticed that this module uses `@property` extensively. Why not putting everything in the `__init__` and check there the type of the values? [related](https://github.com/bigchaindb/bigchaindb/pull/146/files#r56849311)